lab_1_classify_by_unigrams/main.py

```diff
@@ -49,6 +86,20 @@ def compare_profiles(
     :param profile_to_compare: a dictionary of a profile to compare the unknown profile to
     :return: the distance between the profiles
     """
+    if not (
+            isinstance(unknown_profile, dict) and isinstance(profile_to_compare, dict) and 'name' in unknown_profile
+            and 'freq' in unknown_profile and 'name' in profile_to_compare and 'freq' in profile_to_compare
+    ):
+        return None
+
+    else:
+        unknown_profile_freq = unknown_profile['freq']
+        profile_to_compare_freq = profile_to_compare['freq']
+        actual = list(unknown_profile_freq.values())
+        predicted = list(profile_to_compare_freq.values())
+        mse = calculate_mse(predicted, actual)
```

Comment:
Are you sure you're passing the correct lists to the MSE function? 
The lists are supposed to represent the frequencies of the same letters in different languages, meaning that if the first number in the first list designates the frequency of a letter "a" in the first language, for example, then the first number in the second list must represent the frequency of "a" as well, but for the second language.

Reply:
fixed